main.go

```diff
@@ -26,12 +24,7 @@ func main() {
 	}
 
 	if debugMode {
-		err := plugin.Debug(context.Background(), "vmware/tanzu/tmc", opts)
-		if err != nil {
-			log.Fatal(err.Error())
-		}
-
-		return
+		opts.ProviderAddr = "vmware/dev/tanzu-mission-control"
```

Comment:
should we have dev ? what is the advantage of this ProviderAddr  ?

Reply:
This is to test the provider changes locally in debug mode, since in the Makefile we build as `vmware/dev/tanzu-mission-control` using the same here